### ISSUE TITLE
"No result found" for any mistyped address

### ISSUE THREAD

**Issue by rustyrussell** [Blockstream/esplora#134]
If I enter '1RustyRX2oai4EYYDpQGWvEL62BBGqN9T' I get the expected results, but if I mistype or truncate, I get "No result found".

If it looks like an address, but is otherwise malformed, it would be nice to have a diagnostic rather than a generic message as you'd get for an unused address.

**Comment by shesek** [comment]
> If it looks like an address, but is otherwise malformed, it would be nice to have a diagnostic 

Yes, that would make sense. How would you define "looks like an address"? Perhaps anything that uses the base58 alphabet and has over N characters?

> rather than a generic message as you'd get for an unused address.

Note that unused but valid addresses don't return an error message, they give you back the address page with no transactions.
